### PR TITLE
PIM-10717 update with PLRs and string/boolean

### PR DESCRIPTION
**Pull request by felix-couprie**

### CODE REVIEW

**Review by ghoussard — APPROVED** [review]

File: content/md/assets/about-product-link-rules.md [hunk 1]
```diff
@@ -206,13 +206,15 @@ You can use multiple conditions to make your selection. Those conditions are cum
 Here is the list of the fields you can use to select your products:
 
 - Among the product properties:
-  * the `product family`,
-  * the `product categories`,
-  * the `product status`,
+  * the `family`,
+  * the `categories`,
+  * the `status`,
```
Comment (StevenVAIDIE) [inline content/md/assets/about-product-link-rules.md:211]: Status means `enabled` I think 
```suggestion
```

File: content/md/assets/about-product-link-rules.md [hunk 2]
```diff
@@ -206,13 +206,15 @@ You can use multiple conditions to make your selection. Those conditions are cum
...
+  * the `enabled`.
 - Among the product attributes:
   * the `identifier` attribute,
-  * the `text` attributes,
-  * the `simple/multi select` attributes.
+  * the `text` & `text area` attributes,
+  * the `simple select` and `multi select` attributes,
+  * the `yes/no` attribute.
```
Comment (StevenVAIDIE) [inline content/md/assets/about-product-link-rules.md:218]: We can do a selection on every attribute type so I don't know if we need to specify it

File: content/md/assets/about-product-link-rules.md (hunk 1 above)
Comment (ghoussard) [inline content/md/assets/about-product-link-rules.md:211]:
```suggestion
```

**Review by StevenVAIDIE — APPROVED** [review]

File: content/md/assets/about-product-link-rules.md (hunk 2 above)
Comment (felix-couprie) [reply content/md/assets/about-product-link-rules.md:217]: Hmm this is weird, as the documentation is supposed to be exhaustive... ALL attributes are really usable in the selection?

**Pull request merged**